jira_utils.go

```diff
@@ -1,61 +1,22 @@
 package main
 
 import (
-	"bufio"
 	"fmt"
-	"os"
 	"strings"
-	"time"
 
 	bfconfluence "github.com/kentaro-m/blackfriday-confluence"
 	"github.com/michael-go/go-jsn/jsn"
 	bf "gopkg.in/russross/blackfriday.v2"
 )
 
-/***
-function getDate
-return date: string
-argument: none
-return a string containing date and time
-***/
-func getDate() string {
-
-	now := time.Now().Round(0)
-	y := fmt.Sprint(now.Year()) + "_"
-	m := fmt.Sprint(int(now.Month())) + "_"
-	d := fmt.Sprint(now.Day()) + "_"
-	h := fmt.Sprint(now.Hour()) + "_"
-	min := fmt.Sprint(now.Minute()) + "_"
-	s := fmt.Sprint(now.Second())
-
-	return y + m + d + h + min + s
+type Tickets struct {
+	Summary         string `json:Summary`
+	Description     string `json:Description`
+	JiraIssueDetail string `json:jiraDetails`
```

Comment:
this type feels a bit wrong/simplistic.... Why don't we have a type to know we expect key:Array[{something something}] ?

Reply:
That 's just the structure for a ticket not for the full list (after the retry in the openJiraTickets). Maybe should be ticket not tickets. I guess you mean that you expect an other struct with, but I had an issue with that as it seem that the using a structure fix the key name and the key I use is the projects ID.